reference.md

```diff
@@ -45,39 +45,39 @@ root: .
 `rm`
 : remove a file or files. NB: *USE WITH CAUTION!!!*
 
-### Basic Git commands 
+### Basic Git commands
 
 Git cheat sheet handouts:
 
-* https://training.github.com/kit/downloads/github-git-cheat-sheet.pdf 
-* https://www.git-tower.com/blog/git-cheat-sheet/
+* [GitHub](https://github.github.com/training-kit/downloads/github-git-cheat-sheet.pdf)
```

Comment:
Thanks for contributing @wickr. A couple of minor requests: Rather than "GitHub" and "Tower git client" for link text, could you alter these to state "GitHub git cheat sheet" and "Tower git client cheat sheet" respectively?  Thanks!

Reply:
Sure, I actually started with that form but dropped it because of the 'header' right above. But with the links hidden it's not really clear. I adjusted the text for the Tower one since it's still a git cheat sheet, not for the client specifically.

Reply:
Thanks; yeah, agreed it's a little redundant. LGTM, and will merge shortly.